armi/reactor/tests/test_components.py

```diff
@@ -706,6 +803,13 @@ def test_dimensionThermallyExpands(self):
 
 
 class TestRectangle(TestShapedComponent):
+    """Test rectangle shaped component.
+
+    .. test:: Rectangle shaped component
+        :id: T_ARMI_COMP_SHAPES2
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.